.eslintrc.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
                 extensions: ['.js'],
             },
         ],
+        'react/default-props-match-prop-types': 'off',
```

Comment:
@SiAdcock This [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/default-props-match-prop-types.md) requires you to define a component's `propTypes` if you've set `defaultProps` on a component. I thought it confusing to enforce implementing this set of types alongside `flow`. Briefily discussed here: https://github.com/yannickcr/eslint-plugin-react/issues/1593

Reply:
Yeah, let's go with Flow for now